src/Spec.ts

```diff
@@ -503,6 +507,129 @@ export default class Spec {
     }
   }
 
+  private async buildMultipage() {
```

Comment:
Also, the logic for writing files should live in `cli.ts`. This file operates as a library; it shouldn't be doing any IO except using the `fetch` function passed to it.